loopchain/peer/block_manager.py

```diff
@@ -656,27 +639,16 @@ def _handle_exception(e):
 
         try:
             if peer_stubs:
-                my_height, max_height = self.__block_request_to_peers_in_sync(peer_stubs,
-                                                                              my_height,
-                                                                              unconfirmed_block_height,
-                                                                              max_height)
+                self.__block_request_to_peers_in_sync(peer_stubs,
+                                                      my_height,
+                                                      unconfirmed_block_height,
+                                                      max_height)
         except Exception as exc:
             _handle_exception(exc)
             return False
```

Comment:
Need to remove this return line, to ensure this func to end at the single point of truth.

Reply:
No problem.